AAI/AAIConnectProfile.md

```diff
@@ -396,6 +397,12 @@ the Broker.
     presented and/or transformed without misrepresenting the original intent,
     except for accommodating for `exp` timestamps to be represented as
     indicated above.
+    
+3.  An Embedded Token Issuer MAY include the `aud` field to identify the 
+    [Brokers](#term-broker) as the intended audience as specified by 
+    [RFC 7523 Section 3](https://tools.ietf.org/html/rfc7523#section-3).
+    The values within the `aud` field must match the `iss` field of access tokens 
```

Comment:
Suggestion just to make language clear. Given this is a MAY, and then have a sentence here saying "the values ... must match". How about:

"When an `aud` claim is specified, the values within the `aud` claim MUST match the `iss` claim for the Broker that includes the embedded token."